charts/shoot-core/components/charts/vpn-shoot/templates/service.yaml

```diff
@@ -1,19 +1,12 @@
+# TODO(shafeeqes): Delete this Helm chart in a future version.
+---
 {{- if not .Values.reversedVPN.enabled }}
```

Comment:
How to incorporate this in the Golang object?
Similarly the conditions which come in between specs in the chart too.

Reply:
?

Reply:
I see that you have introduced`ReversedVPNEnabled bool` field in the `Values`. If it is set to `true` then you add the `Service` to the `registry`, if it is set to false then you don't add it.